wiki/How_You_Can_Help!/en.md

```diff
@@ -1,80 +1,78 @@
+- Look through or search the [Pending Beatmaps](https://osu.ppy.sh/beatmapsets?m=-1&s=4) listing for any artists or songs you like (or just choose at random).
+- Play the beatmaps and leave feedback in threads.
+- Read the [Modding Guide](/wiki/Modding "Modding") and the [Ranking Criteria](/wiki/Ranking_Criteria "Ranking Criteria") to know the basics of technical modding and what makes a beatmap rankable.
+- Join the `#modhelp` and `#modreqs` on in-game chat and join in the conversation (or just follow along).
+- If you have the time and dedication, make a [Modding Queue](/community/forums/60) and have people start coming to you with requests.
 
-While we have set teams that handle the approval process, there's nothing stopping anyone from helping out. It starts as easy as posting in pending beatmap threads leaving your positive feedback or constructive criticism. When you get good, there's no reason you can't join up on one of the approval teams :).
+## Join some user-run projects and contribute to making them happen (or try to make them better if they're active)
 
-### Where to start:
+osu! has a thriving community of technical and competitive people. They range from tinkerers, to seasoned rhythm game veterans. There's a lot of proposals and projects being made exclusively for osu! and you can help them, or drive the future of osu! with your own creation!
 
--   Look through (or search) the [Pending Beatmaps](https://osu.ppy.sh/beatmapsets?m=-1&s=4) listing for artists/songs you like (or just choose ones at random).
--   Play the beatmaps and leave feedback in threads.
--   Read the [Modding Guide](/wiki/Modding "Modding") and learn the basics of more technical modding.
--   Read the [Ranking Criteria](/wiki/Ranking_Criteria "Ranking Criteria") and become familiar with what makes a map rankable.
--   Join `#modhelp` and `#modreqs` on in-game chat and join in the conversation (or just follow along).
--   If you have the time and dedication, making a [Modding Queue](/community/forums/60) thread and have people start coming to you with requests.
+Here's a few things you can do to help:
 
-## Join some user-run projects and contribute to making them happen (or making them better if they are already actively running)!
+- Browse the [torunaments](/community/forums/55) forum. Tournaments takes time and a lot of manpower to organize without hitches, and you can help out by being a spectator judge.
+- Check out what's going on in [General Development](/community/forums/2). This low activity forum is where the [most](/community/forums/topics/58855) [serious](/community/forums/topics/58855) [projects](/community/forums/topics/80027) happen.
+- Join a [beatmap project](/community/forums/53) and learn how fun collaborating with others can be!
+- Start your own project! Whether its just bringing back dancing pippi, or just making a cool thing for osu!, anything is accepted.
+- Bring [osu!monthly](/community/forums/topics/75017) to life.
```

Comment:
From what I can see, the original osu!monthly has been dead for a long time. Even the reboot (http://osumonthly.com/) hasn't released for a year now, so you can probably remove this point.

If you want to include this anyways, you should probably say "Bring ... **back** to life"